added in suggestions from feedback (#8)
madetech/data-101: modules/core/Python.md

## Simple setup:

> When working on a python project it is beneficial to use virtual environments to ensure that all developers on the project have the same dependencies and are able to spin up an environment quickly. A local python environment will allow you install packages to a local environment that won't interfer with your global installed packages.

Reviewer: ```suggestion
When working on a python project it is beneficial to use virtual environments to ensure that all developers on the project have the same dependencies and are able to spin up an environment quickly. A virtual python environment segregates project specific packages from your global installed packages.
```
